test/e2e/snaps/test-snap-update-component.spec.js

```diff
@@ -131,6 +131,15 @@ describe('Test Snap update via snaps component', function () {
           tag: 'button',
         });
 
+        await driver.clickElement({
+          text: 'Install',
```

Comment:
Is there a reason to not use `await driver.clickElement('.mm-checkbox__input');` as we do elsewhere in the test?

Reply:
no particular reason, but I can change it if this is the preferred selector :+1:

Reply:
Yes please! Otherwise LGTM!

Reply:
Oop, I modified my comment, do we not need to wait for the selector first?

Reply:
we don't need to wait since in the `clickElement`method we already wait for the element to be visible and clickable

![Screenshot from 2024-05-31 14-05-31](https://github.com/MetaMask/metamask-extension/assets/54408225/da4e353c-e76c-4e01-ac6e-d678c86626ce)

![Screenshot from 2024-05-31 14-05-09](https://github.com/MetaMask/metamask-extension/assets/54408225/109cca58-143f-4325-b32c-7aeceefd7ba5)

Reply:
👍